devices/IFrameTransformToIWear/src/IFrameTransformToIWear.cpp

```diff
@@ -0,0 +1,518 @@
+/*
+ * Copyright (C) 2021 Istituto Italiano di Tecnologia (IIT)
+ * All rights reserved.
+ *
+ * This software may be modified and distributed under the terms of the
+ * GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#include "IFrameTransformToIWear.h"
+
+#include <yarp/dev/IFrameTransform.h>
+#include <yarp/os/LogStream.h>
+#include <yarp/sig/Vector.h>
+
+#include <map>
+#include <mutex>
+#include <string>
+#include <vector>
+
+const std::string DeviceName = "IFrameTransformToIWear";
+const std::string LogPrefix = DeviceName + " :";
+
+using namespace wearable;
+using namespace wearable::sensor;
+using namespace wearable::devices;
+
+struct ParsedOptions
+{
+    wearable::WearableName wearableName;
+    wearable::sensor::SensorType wearableSensorType;
+    std::string rootFrameID;
+
+    std::vector<std::string> frameIDs;
+};
```

Comment:
This is a plugin and it should not be a problem, but as good practice I'd put this struct in a unique namespace (using ::detail or ::private for example. Nasty bugs in general can happen when multiple libraries are loaded in a single application and they define the same symbol even in their cpp. In the past I cursed a lot when it happened to me :)